### ISSUE TITLE
Lieux-dits en double

### ISSUE THREAD

**Issue by gendy54** [osm-fr/osm-vs-fantoir#419]
Hello,
https://bano.openstreetmap.fr/pifometre/index.html?insee=58086

LD Les Carrieres (avec l'oubli d'accent)

Je me disais tiens, c'est curieux, ce LD est présent 4 fois.
Hors c'est 2 fois 2 LD différents mais avec le même Fantoir:

1. https://bano.openstreetmap.fr/pifometre/pifomap.html?insee=58086&ratio=no#map=18/47.429377/2.972094
2. https://bano.openstreetmap.fr/pifometre/pifomap.html?insee=58086&ratio=no#map=18/47.396564/2.933823

![image](https://github.com/user-attachments/assets/a7cd05ab-3b39-4c74-9862-3890b1df5055)

**Comment by vdct** [comment]
Oui, pas de miracle ici, il y a 2 LD homonymes sur l'emprise de la commune et une seule entrée TOPO...
